dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java

```diff
@@ -27,8 +27,6 @@
 })
 public class FATSuite {
     @ClassRule
-    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly())
-                    .andWith(FeatureReplacementAction.EE8_FEATURES().removeFeature("jaxws-2.2").addFeature("jaxws-2.3").withID("jaxws-2.3"))
-                    .andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("jaxws-2.3"));
+    public static RepeatTests r = RepeatTests.with(new EmptyAction()).andWith(FeatureReplacementAction.EE9_FEATURES());
```

Comment:
did you intend to remove the `.fullFATOnly()` from the EmptyAction?

Reply:
Ah, yeah I do. I want both `jaxws-2.2` and `EE9` tests to run in lite mode for now.